bench/workgen/runner/cache_workload_dirty_trigger.py

```diff
@@ -196,8 +222,25 @@ def get_cache_eviction_stats(session, cache_eviction_file):
 # The new threads will also be added to the workload below.
 ############################################################################
 
-# 15% update workload
-cache_workload = Workload(context, 15 * thread_upd10k_sleep10 + 85 * thread_read10k_sleep10)
+#Configure only the expected % of read operations
+#15% update workload to hit the cache dirty trigger threshold
+read_ops = 85
+#Calculate the write operations based on configured write operations.
+write_ops = (100 - read_ops)
```

Comment:
The write_ops variable can be removed, as it doesn't appear to be referenced anywhere.